packages/chips/chip-set/index.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import {MDCChipSetFoundation} from '@material/chips';
+
+import Chip from '../chip';
+
+export default class ChipSet extends Component {
```

Comment:
In the `README`'s example, you use a different import style for `Component`, e.g.: 

```
import React from 'react';

class Foo extends React.Component {...}
```

Suggest changing the example in the `README` to use this code style instead.